02-wrangling.Rmd

```diff
@@ -2,14 +2,14 @@
 
 ## Overview 
 
-This chapter will be centered around tools for cleaning and wrangling data that move data from its raw format into a format that is suitable for data analysis. They 
-will be presented in the context of a real world data science application, providing more practice working through a whole case study.
+This chapter will be centred around tools for cleaning and wrangling data that move data from its raw format into a suitable format for data analysis. They will be presented in the context of a real-world data science application, providing more practice working through a whole case study.
 
 ## Chapter learning objectives
 By the end of the chapter, students will be able to:
 
 * define the term "tidy data"
-* discuss the advantages and disadvantages from storing data in a tidy data format
+* discuss the advantages and disadvantages of storing data in a tidy data format
+* contrast the basic data types and structures in R
```

Comment:
We should be more specific here since we are really focusing on just vectors and data frames.
```suggestion
* define what vector and data frames are in R, and describe how they relate to each other
* describe the types of data that can be stored in integer, double, character, logical, and factor vectors in R
```